src/test/java/pageobject_model/page/CloudGoogleCalculatorPage.java

```diff
@@ -0,0 +1,162 @@
+package pageobject_model.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pageobject_model.test.Helpers;
+
+
+public class CloudGoogleCalculatorPage extends BasePage {
+
+    @FindBy(xpath = "//a[@href='https://cloud.google.com/pricing/calculators']")
+    private WebElement calculatorsTab;
+
+    @FindBy(xpath = "//input[@id='input_52']")
+    private WebElement instancesNumberField;
+
+    @FindBy(xpath = "//md-select[@id='select_75']")
+    private WebElement machineTypeDropdown;
+
+    @FindBy(xpath = "//md-option[@value='CP-COMPUTEENGINE-VMIMAGE-N1-STANDARD-8']")
+    private WebElement machineTypeDropdownElement;
+
+    @FindBy(css = "md-input-container > [aria-label='Add GPUs']")
+    private WebElement addGpuCheckbox;
+
+    @FindBy(xpath = "//input[@id='input_52']")
+    private WebElement instancesNumberFieldText;
+
+    @FindBy(xpath = "//md-select[@placeholder='Number of GPUs']/md-select-value")
+    private WebElement gpuNumberDropdown;
+
+    @FindBy(xpath = "//*[@class='md-select-menu-container md-active md-clickable']//md-option[@value='1']")
+    private WebElement gpuNumberDropdownElement;
+
+    @FindBy(xpath = "//md-select[@placeholder='GPU type']/md-select-value")
+    private WebElement gpuTypeDropdown;
+
+    @FindBy(xpath = "//*[@value='NVIDIA_TESLA_V100']")
+    private WebElement gpuTypeDropdownElement;
+
+    @FindBy(xpath = "//md-select[@placeholder='Local SSD']/md-select-value")
+    private WebElement localSsdDropdown;
+
+    @FindBy(xpath = "//div[@class='md-select-menu-container md-active md-clickable']//md-option[@value='2']")
+    private WebElement getLocalSsdDropdownElement;
+
+    @FindBy(xpath = "//md-select[@placeholder='Datacenter location']/md-select-value")
+    private WebElement dataCenterDropdown;
+
+    @FindBy(xpath = "//*[@class='md-select-menu-container md-active md-clickable']//*[@value='europe-west3']")
+    private WebElement dataCenterDropdownElement;
+
+    @FindBy(xpath = "//md-select[@placeholder='Committed usage']/md-select-value")
+    private WebElement committedUsageDropdown;
+
+    @FindBy(xpath = "//*[@class='md-select-menu-container md-active md-clickable']//*[@value='1']")
+    private WebElement committedUsageDropdownElement;
+
+    @FindBy(xpath = "//button[@class='md-raised md-primary cpc-button md-button md-ink-ripple']")
+    private WebElement submitButton;
+
+    public CloudGoogleCalculatorPage(WebDriver driver) {
+        super(driver);
+        initElements();
+    }
+
+    protected BasePage initElements() {
+        PageFactory.initElements(driver, this);
+        return null;
+    }
+
+    public CloudGoogleCalculatorPage openPage() {
+        Helpers.waitUntilClickable("//a[@href='https://cloud.google.com/pricing/calculators']", driver);
```

Comment:
It will be better if you will store all locator in the class header, if you start in such way. If it's not a 'findBy' annotation, you could initialise String variable with expected locator.